### PR TITLE
Mbasiuk/tdi 40066 snowflake salesforce repository refresh issue

### PR DESCRIPTION
**Pull request by mbasiuk-talend**
**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TDI-40066
https://jira.talendforge.org/browse/TUP-19728
https://jira.talendforge.org/browse/TUP-19710

**What is the new behavior?**
Created a workaround for Snowflake/Salesforce that prevents from triggering API calls for retrieving table/module metadata.

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by mbasiuk-talend** [comment]
AbstractSchemaController.java was containing Windows line ending.

File: main/plugins/org.talend.designer.core/src/main/java/org/talend/designer/core/ui/editor/properties/controllers/AbstractSchemaController.java [hunk 1]
```diff
@@ -1217,7 +1217,8 @@ protected Command createComboCommand(CCombo combo) {
                 IElementParameter repositorySchemaType = param.getParentParameter().getChildParameters()
```
Comment (mbasiuk-talend) [inline main/plugins/org.talend.designer.core/src/main/java/org/talend/designer/core/ui/editor/properties/controllers/AbstractSchemaController.java]: This file contains Windows style line endings, will be changed after in other PR.

File: main/plugins/org.talend.designer.core/src/main/java/org/talend/designer/core/ui/editor/cmd/ChangeValuesFromRepository.java [hunk 2]
```diff
@@ -381,10 +389,13 @@ public void execute() {
...
                         if (this.moduleUnit != null) {
-                            objectValue = moduleUnit.getModuleName();
-                        } else {
-                            objectValue = null;
-                        }
+                             objectValue = moduleUnit.getModuleName();
+                         } else {
+                             objectValue = null;
+                         }
+                    } else if ("table.tableName".equals(repositoryValue)) {
```
Comment (wwang-talend) [inline main/plugins/org.talend.designer.core/src/main/java/org/talend/designer/core/ui/editor/cmd/ChangeValuesFromRepository.java]: better to put it together with "module.moduleName" check for reading, not important, keep this now

Comment (mbasiuk-talend) [reply main/plugins/org.talend.designer.core/src/main/java/org/talend/designer/core/ui/editor/cmd/ChangeValuesFromRepository.java]: Yes, you're right. It seems this was the first changes I've made, and others have such conditions.

**Pull request merged**